### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by kylepjohnson**
From the Makefile:

```
test:
	poetry run tox

testOnlyDocTests:
	poetry run pytest --doctest-modules --cov-report term-missing --cov-report html:htmlcov --cov=src/cltkv1 src/cltkv1/ --ignore=tests

testOnlyTestsDir:
	poetry run pytest --disable-warnings tests
```

`testOnlyDocTests` takes 24 mins with the models already downloaded.

Let's see how it runs on the server … might not be much faster than before.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ before_script:
 #  - poetry run pre-commit run --all-files
 
 script:
+  - make testOnlyDocTests
```
Comment (todd-cook) [inline .travis.yml:27]: You probably mean to remove the `make test` as well

Comment (kylepjohnson) [reply .travis.yml:27]: Yeah, I haven't thought through this part yet. I will try these or some variation with the build matrix.

**Comment by kylepjohnson** [comment]
Refactored doctests all pass: https://travis-ci.org/github/cltk/cltkv1/builds/701420862

Going to try the full suite next

**Comment by kylepjohnson** [comment]
Here's what the matrix build looks like, currently first passed second running: https://travis-ci.org/github/cltk/cltkv1/builds/701438712

**Comment by kylepjohnson** [comment]
Things looking like all will: pass https://travis-ci.org/github/cltk/cltkv1/builds/701488078

**Pull request merged**